src/Meilisearch/Key.cs

```diff
@@ -9,6 +9,15 @@ namespace Meilisearch
     /// </summary>
     public class Key
     {
+        /// <summary>
+        /// Create a key whith an expiry date. Pass null if it's not expiring.
+        /// </summary>
+        /// <param name="expiresAt">Expiry date.</param>
+        public Key(DateTime? expiresAt)
```

Comment:
I read your comment regarding this change, and I understood your intentions about that, but I am not sure if this is the way to go. I like the approach to make it harder to user make a mess, but you know they still, can make a mess. 

Given that we released v0.10 a few weeks ago, I think we could wait for more feedback before introducing a new breaking change.

> Why not having an ExpiresAt value far in the future for this instead of a null values ? JsonSerializer can be tricky for other clients with strongly typed languages.

I partially agree because I don't like the idea to have a key that never expires, this always triggers a security concern. But since we have this option, I agree with the idea to have the null value to represent it.

I invite you to open a discussion here: https://github.com/meilisearch/product/discussions regarding this topic, I'll be glad to share my concerns there :)